tmt/steps/execute/__init__.py

```diff
@@ -55,77 +58,199 @@
 # Metadata file with details about the current test
 TEST_METADATA_FILENAME = 'metadata.yaml'
 
-# Scripts source directory
+#: Scripts source directory
 SCRIPTS_SRC_DIR = tmt.utils.resource_files('steps/execute/scripts')
 
+#: The default scripts destination directory
+SCRIPTS_DEST_DIR = Path("/usr/local/bin")
+
+#: The default scripts destination directory for rpm-ostree based distributions, https://github.com/teemtee/tmt/discussions/3260
+SCRIPTS_DEST_DIR_OSTREE = Path("/var/lib/tmt/scripts")
```

Comment:
Maybe we should call these two `DEFAULT_*`, because the actual path will come out of `effective_scripts_dest_dir()`.

Reply:
added as DEFAULT